### PR TITLE
Update 21-1.cgel

### PR DESCRIPTION
**Pull request by mebwells**

### CODE REVIEW

**Comment by bwaldon** [comment]
- Possessive PP “of the assisting personnel” should be Mod, not Comp (see, e.g., `newsgroup-groups.google.com_FOOLED_1bf9cdc5a4c2ac48_ENG_20050904_130400-0026` in EWT dataset) 
- Are there arguments for “merely” as part of Mod to VP? I think “merely a subterfuge…” passes (at least one) constituency test: “Merely a subterfuge, it was not.” “*A subterfuge, it was not merely.” 
- Lowercase “T” in “The” 
- Remove “_” in “as_to” 
- Add lemma and POS tags in the .cgel file

**Comment by mebwells** [comment]
I agree about "merely" according to your constituency test as well and have changed the structure accordingly, as well as fixing the other errors. Thanks!

File: datasets/oneoff/21-1.cgel [hunk 1]
```diff
@@ -9,10 +9,10 @@
         :Head (Nom
             :Head (N :t "court")))
     :Head (VP
-        :Head (V_aux :t "may")
+        :Head (V_aux :t "may" :l "may" :xpos "MD")
```
Comment (nschneid) [inline datasets/oneoff/21-1.cgel:12]: We use `:l` only for lemmas that differ from the `:t` token

Comment (mebwells) [reply datasets/oneoff/21-1.cgel:12]: oops... thanks

**Pull request merged**